golang/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: golan Helm Chart
 name: golang
-version: 8.0.7
+version: 8.1.0
```

Comment:
Sett til versjon `9.0.0` da dette er en ikke-bakoverkompatibel endring for de som allerede bruker `.Values.service.targetPort`.